### ISSUE TITLE
constrained property optimization

### ISSUE THREAD

**Issue by giame** [DeepGraphLearning/torchdrug#100]
Hello,
Is there a way to reproduce the Constrained property optimization experiments from GCPN and GraphAF on torchdrug?
Similarly to the property optimization from https://torchdrug.ai/docs/tutorials/generation.html

Thanks!

**Comment by shichence** [comment]
Yes, you can. The only difference between Constrained property optimization and property optimization is that we start from an existing subgraph in the former case. To implement it, you basically need to modify the part of codes that handle the graph initialization for training and inference.
